bindings/pyroot/pythonizations/test/CMakeLists.txt

```diff
@@ -118,7 +118,9 @@ endif()
 
 # RTensor pythonizations
 if (dataframe AND tmva)
-    ROOT_ADD_PYUNITTEST(pyroot_pyz_rtensor rtensor.py PYTHON_DEPS numpy)
+    if(NOT MSVC OR "${CMAKE_GENERATOR_PLATFORM}" MATCHES "Win32" OR win_broken_tests)
+        ROOT_ADD_PYUNITTEST(pyroot_pyz_rtensor rtensor.py PYTHON_DEPS numpy)
```

Comment:
So then we give up on this one? No way to solve it on Win64?

Reply:
I want to enable nightlies on Win64, and don't want to wait until all tests are fixed. That doesn't mean I'm giving up :wink: